src/Raku/ast/variable-declaration.rakumod

```diff
@@ -726,10 +731,12 @@ class RakuAST::VarDeclaration::Simple
                     }
 
                     my $value := $expression.compile-time-value;
-                    unless nqp::istype($value,$vartype) {
+                    if !nqp::istype($value,$vartype) &&
+                        # nqp::istype($vartype,Numeric) returns 0 here, so check all relevant types
+                        (nqp::istype($vartype,Int) || nqp::istype($vartype,Num) || nqp::istype($vartype,Rat) || nqp::istype($vartype,FatRat)) {
                         $resolver.add-sorry: $resolver.build-exception:
-                          'X::Syntax::Number::LiteralType',
-                          :varname(self.name), :$vartype, :$value
+                            'X::Syntax::Number::LiteralType',
```

Comment:
trailing whitespace?

Reply:
looks like indent = 2 vs indent = 4 to me